src/javascript/binary/pages/portfiliows.js

```diff
@@ -0,0 +1,199 @@
+var PortfolioWS =  (function() {
+
+    'use strict';
+
+    var pageLanguage = page.language();
+    if(!pageLanguage) pageLanguage = "EN";
+
+    var rowTemplate;
+    var retry;
+
+    var init = function() {
+        // get the row template and then discard the node as it has served its purpose
+        rowTemplate = $("#portfolio-dynamic tr:first")[0].outerHTML;
+        $("#portfolio-dynamic tr:first").remove();
+        BinarySocket.send({"balance":1});
+    };
+
+
+    /**
+     * Show balance
+    **/
+    var updateBalance = function(data) {
+        $("span[data-id='balance']").text(StringUtil.formatCurrency(data.balance.balance, data.balance.currency));
+        if(parseFloat(data.balance.balance, 10) > 0) {
+            $("#if-balance-zero").remove();
+        }
+        BinarySocket.send({"portfolio":1});
+    };
+    
+    /**
+     * Updates portfolio table
+    **/
+    var updatePortfolio = function(data) {
+
+        /**
+         * Check for error
+        **/
+        if("error" in data) {
+            throw new Error("Trying to get portfolio data, we got this error", data.error);
```

Comment:
This is not shown in the interface, just in console. Do we need localization for it?